.circleci/codeReview.yml

```
@@ -0,0 +1,111 @@
+  pr-review-with-tracking:
+    docker:
+      - image: cimg/python:3.11-node
+    resource_class: medium
+    steps:
+      - checkout
+      
+      # Install Claude Code CLI 
+      - run:
+          name: Install Claude Code CLI
+          command: |
+            # Use the native installer like the docs recommend 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)